web/src/main/java/se/inera/intyg/webcert/web/service/notification/NotificationRedeliveryStatusUpdateCreatorService.java

```diff
@@ -74,10 +64,10 @@
     /**
      * Creates a {@link CertificateStatusUpdateForCareType} based on the information received in the {@link NotificationRedelivery}.
      */
-    public CertificateStatusUpdateForCareType createCertificateStatusUpdate(NotificationRedelivery redelivery, Handelse event)
-        throws IOException, ModuleException, ModuleNotFoundException, TemporaryException {
+    public String createCertificateStatusUpdate(NotificationRedelivery redelivery, Handelse event)
```

Comment:
Maybe rename the method to something more descriptive. Also update the javadoc.

Also look at the private methods.. with the change to manage it as an xml-string the method´s change purpose and for readability should be renamed.

Reply:
Thanks for valuable input. I agree and will implement the suggestions.